tools/patch/patch.py

```diff
@@ -513,6 +513,8 @@ def normalize_backend(backend):
         return "omniinfer"
     elif input_lower in ["verl"]:
         return "verl"
+    elif input_lower in ["robo", "robotics"]:
+        return ["Megatron-LM", "Megatron-Energon"]
```

Comment:
Robotics is a domain-specific concept that should be distinguished from the backend, with domain design incorporated.